src/OrbitBase/ExecutablePathLinux.cpp

```diff
@@ -27,7 +27,7 @@ std::filesystem::path GetExecutablePath() {
   return std::filesystem::path(std::string(buffer, length));
 }
 
-ErrorMessageOr<std::filesystem::path> GetExecutablePath(int32_t pid) {
+ErrorMessageOr<std::filesystem::path> GetExecutablePath(uint32_t pid) {
   char buffer[PATH_MAX];
```

Comment:
check that pid is < MAX_INT

Reply:
reverted to using int32_t

Reply:
Shouldn't we then use `pid_t` instead of `int32_t`?

Reply:
yep done.